zerver/lib/notifications.py

```diff
@@ -4,7 +4,7 @@
 from confirmation.models import Confirmation, create_confirmation_link
 from django.conf import settings
 from django.template import loader
-from django.utils.timezone import now as timezone_now
+from django.utils.timezone import localtime, now as timezone_now
```

Comment:
The localtime import is not needed, right?

Reply:
Yeah you're right, I will remove it.